tool/templates/c/impl.h.jinja

```diff
@@ -1,6 +1,13 @@
 {% if is_for_cpp -%}
 extern "C" {
 {%- endif %}
+{% for cb_and_struct_def in cb_structs_and_defs %}
+typedef struct {{ cb_and_struct_def.name }} {
+    const void* data;
+    {{ cb_and_struct_def.return_type }} (*run_callback)(const void*{% if cb_and_struct_def.params_types != "" %}, {{ cb_and_struct_def.params_types }} {% endif %});
+    void (*destructor)(const void*);
+} {{ cb_and_struct_def.name }};
+{% endfor %}
```

Comment:
```suggestion
{%- endfor %}
```

probably this will fix your whitespace woes

Reply:
thank you, i was trying to track this down!